### ISSUE TITLE
[a11y] TW menubar creates an anonymous "navigation" "navigation" landmark. It should be "top menu bar" 

### ISSUE THREAD

**Issue by pmario** [Jermolene/TiddlyWiki5#7395]
Trying to make the breadcrumbs useful for screen readers, I found out that the menu-bar has no aria-label

**To reproduce on windows using Narrator**

- activate narrator
- narrator-key + F5   ... list all landmarks
- the menu-bar creates "navigation" "navigation"   speech output.

![grafik](https://user-images.githubusercontent.com/374655/230387529-17bbc818-3a44-4a42-9e5d-64cf18a1d673.png)

It should be `aria-label="top menu bar"` or something similar

just a reminder
